atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/SweepTaskRunnerImpl.java

```diff
@@ -73,6 +75,7 @@
  */
 public class SweepTaskRunnerImpl implements SweepTaskRunner {
     private static final Logger log = LoggerFactory.getLogger(SweepTaskRunnerImpl.class);
+    private static final int MAX_VALUES_TO_DELETE_BATCH = 10_000;
```

Comment:
Should we make this a part of config, in case even 10k causes delete to blow up?